optimum/intel/openvino/quantization.py

```diff
@@ -119,7 +121,8 @@ def quantize(
         self,
         calibration_dataset: Dataset = None,
         save_directory: Union[str, Path] = None,
-        quantization_config: OVConfig = None,
+        quantization_config: QuantizationConfigMixin = None,
+        ov_config: OVConfig = None,
```

Comment:
Not sure we need both an `ov_config` and `quantization_config`, would prefer to keep only one if that's feasible

Reply:
@echarlaix, how about deprecating `ov_config` here and migrating to different types of `QuantizationConfigMixin` so that there will be only `quantization_config` parameter?

Reply:
was actually thinking about the opposite as `ov_config` have a quantization section, need to think a bit about this